platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gui/GuiEventStorage.java

```diff
@@ -45,19 +45,23 @@ public class GuiEventStorage {
      * Creates an empty instance
      *
      * @param configuration this node's configuration
-     * @param roster   the network's roster
+     * @param roster the network's roster
      */
     public GuiEventStorage(@NonNull final Configuration configuration, @NonNull final Roster roster) {
 
         this.configuration = Objects.requireNonNull(configuration);
         final PlatformContext platformContext = PlatformContext.create(configuration);
 
-        this.consensus = new ConsensusImpl(platformContext, new NoOpConsensusMetrics(), roster);
+        final ConsensusConfig consensusConfig =
+                platformContext.getConfiguration().getConfigData(ConsensusConfig.class);
+        this.consensus =
+                new ConsensusImpl(consensusConfig, platformContext.getTime(), new NoOpConsensusMetrics(), roster);
```

Comment:
Nit: I believe this is the third time that I see this. How about passing `configuration` to the constructor of `ConsensusImpl` and extract `ConsensusConfig` there?